README.md

```diff
@@ -62,6 +71,12 @@ QIT currently supports the following types of tests:
 - Integrate QIT as part of your pull requests with our example [GitHub Workflows integration](https://woocommerce.github.io/qit-documentation/#/workflows/getting-started).
 - Create and share application passwords with your development team, allowing them to use the QIT CLI without having full access to your WooCommerce.com account.
 
+## Is this tool exclusive to WooCommerce.com Partner Developers?
+
+QIT is an innovative initiative aimed at enhancing the accessibility of automated testing for developers. Our ultimate aspiration is to extend the benefits of this tool to a wider circle of WordPress developers, thereby strengthening the entire ecosystem.
+
+However, as we find ourselves in the preliminary phase of this project, we are currently extending this offering solely to our WooCommerce.com Partner Developers. If you're eager to utilize QIT and want to voice your interest, we welcome you to [submit an issue](https://github.com/woocommerce/qit-cli/issues) in our GitHub repository. We deeply value your feedback as it plays a critical role in shaping the future of our project.
+
```

Comment:
Wording suggestion, I think we can start by selling the tool then move into how we want to further expand it out. So maybe for this section we can say:

```
QIT is an innovative initiative aimed at enhancing the accessibility of automated testing for developers.

In this initial phase, we've started with bringing this to Woo Marketplace Partner Developers. Our ultimate aspiration is to extend the benefits of this tool to a wider circle of WordPress developers, thereby strengthening the entire ecosystem.

If you're eager to utilize QIT and want to voice your interest, we welcome you to [submit an issue](https://github.com/woocommerce/qit-cli/issues) in our GitHub repository. We deeply value your feedback as it plays a critical role in shaping the future of our project.
```

Let me know what you think.

Reply:
Thanks, changed